### PR TITLE
UI Text changes for DP-30 Toc View add note and select chapter popup …

### PR DESCRIPTION
**Pull request by Ekta29yadav**
…for sourcing

### CODE REVIEW

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.html [hunk 1]
```diff
@@ -492,8 +492,8 @@ <h5 class="py-15 px-20 font-weight-bold fs-0-92 text-capitalize">
 												</small>
 											</div>
 											<div class="four wide column px-0 mt-5">
-												<div class="sb-color-gray-400 fxsmall text-lowercase">{{tempCollection.selected}} out of {{tempCollection.total}} chapters selected</div>
-												<div class="sb-color-primary fxsmall cursor-pointer" (click)="chooseChapters(tempCollection);">
+												<div class="sb-color-gray-400 fxsmall text-lowercase">{{tempCollection.selected}} out of {{tempCollection.total}} chapter(s) selected</div>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/create-program.component.html:495]: Why not used resource bundle string???

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html [hunk 2]
```diff
@@ -96,6 +96,9 @@
       <!-- Accordion Section -->
       <div class="sb-pageSection-content sb-bg-white py-24 px-0 dynamicHeight">
         <div class="ui container sb-textbook-details-container mb-24">
+          <div class="d-flex flex-jc-flex-end fs-0-785 font-italic sb-color-gray-300 mb-15">
+            <span class="w-30 text-right">{{resourceService.frmelmnts.lbl.noteYouMayNotSeeAll}}</span>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:100]: Don't we want to show this note to the admins and individual contributors while nominating for a program?

Comment (Ekta29yadav) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:100]: As per kamesh comment he mentioned Note needs to be everywhere in TOC view
you can check comment here [](https://project-sunbird.atlassian.net/browse/DP-30)

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.html [hunk 3]
```diff
@@ -492,8 +492,8 @@ <h5 class="py-15 px-20 font-weight-bold fs-0-92 text-capitalize">
 												</small>
 											</div>
 											<div class="four wide column px-0 mt-5">
-												<div class="sb-color-gray-400 fxsmall text-lowercase">{{tempCollection.selected}} out of {{tempCollection.total}} chapters selected</div>
-												<div class="sb-color-primary fxsmall cursor-pointer" (click)="chooseChapters(tempCollection);">
+												<div class="sb-color-gray-400 fxsmall text-lowercase">{{tempCollection.selected}} {{resource.frmelmnts.lbl.outOf}} {{tempCollection.total}} {{resource.frmelmnts.lbl.chaptersSelected}}</div>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/create-program.component.html:495]: frmelmnts.lbl.nCharactersLeft={noOfChars} Character(s) left

In this string {noOfChars} is the dynamic thing, which they are replacing like this

{{ resourceService.frmelmnts.lbl.nCharactersLeft | interpolate:'{noOfChars}': titleCharacterLimit - (editTitle.length > titleCharacterLimit ? titleCharacterLimit : editTitle.length) }}

Can you try using same? @Ekta29yadav

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html (hunk 2 above)
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:100]: please change this variable noteYouMayNotSeeAll to partialTextbookSelectionNote

Comment (twsvaishali) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html:100]: @Ekta29yadav

File: src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html [hunk 4]
```diff
@@ -152,8 +152,8 @@ <h4 class="sb-pageSection-title">
       [telemetryInteractCdata]="telemetryInteractCdata"
       [telemetryInteractPdata]="telemetryInteractPdata">{{ resourceService?.frmelmnts?.lbl?.open }}</button>
                   </td>
-                  <td *ngIf="issourcingOrgAdmin && this.router.url.includes('/sourcing')" class="fs-1-5 sb-dock-icon-tooltip" >
-                    <i class="icon edit sb-color-primary mx-10 cursor-pointer ui" data-tooltip="Modify" (click)="editOnClick(program)"></i>
+                  <td *ngIf="issourcingOrgAdmin && this.router.url.includes('/sourcing')" class="fs-1-5 sb-dock-icon-tooltip px-5">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html:155]: what issue this fix is fixing? @Ekta29yadav

Comment (Ekta29yadav) [reply src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.html:155]: Sometime in sourcing program list view edit and delete icon is not aligned properly due to resolution issue.So this fix is for that only.

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html (hunk 2 above)
Comment (Ekta29yadav) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.html]: done

**Pull request merged**